pkg/model/clusteradministratorentity.go

```diff
@@ -0,0 +1,54 @@
+package model
+
```

Comment:
Maybe we can handle it in the same way that we just store the JSON and don't use dedicated entities. But let's discuss that what we need and what wold be simpler for us.